website/static/css/timeline.css

```diff
@@ -0,0 +1,51 @@
+.timeline-container {
+  max-width: 800px;
+  margin: auto;
+}
+
+.timeline-event {
+  margin: 0;
+
+  position: relative;
+  display: flex;
+
+  text-align: left;
+}
+
+.timeline-event::before {
+  content: "";
+  background-color: #ffd533;
+  position: absolute;
+  bottom: 0;
+  top: 0;
+  left: 1.5rem;
+  width: 3px;
+}
+
+.timeline-event-dot {
```

Comment:
Maybe we can shrink the dot when it is `:empty`.

Reply:
I don't like it because it seems to visually communicate that it's "less important", while I didn't put the date just because `@babel/eslint-parser` and `babel-polyfills` have been two efforts continued over time without a specific date.